package.json

```diff
@@ -8,7 +8,7 @@
 		"clean": "rimraf .next node_modules",
 		"lint": "next-hashicorp lint",
 		"lint:stylelint": "npx stylelint '**/*.css'",
-		"prebuild": "hc-tools ./scripts/remove-inactive-product-pages.ts && hc-tools ./scripts/generate-opt-in-redirect-checks.ts",
+		"prebuild": "hc-tools ./scripts/remove-inactive-product-pages.ts && hc-tools ./scripts/generate-opt-in-redirect-checks.ts && hc-tools ./scripts/generate-tutorial-variant-map.ts",
```

Comment:
We should add a `prestart` script as well that runs this, otherwise I think the dev server would error if the file isn't there